bsdd-graphql-soml-orig.yaml

```diff
@@ -0,0 +1,1149 @@
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionThermodynamicTemperature:
+        name: dimensionThermodynamicTemperature
+        label: The Thermodynamic temperature value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionAmountOfSubstance:
+        name: dimensionAmountOfSubstance
+        label: The Amount of substance value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dimensionLuminousIntensity:
+        name: dimensionLuminousIntensity
+        label: The Luminous Intensity value of the dimension
+        range: int
+        max: 1
+        nonNullable: false
+      dynamicParameterPropertyCodes:
+        name: dynamicParameterPropertyCodes
+        label: List of codes of the properties which are parameters of the function
+          for a dynamic property. Only applicable for dynamic properties (IsDynamic)
+        range: string
+        max: inf
+        nonNullable: false
+      example:
+        name: example
+        label: Illustrate possible use or values of the Property
+        range: string
+        max: 1
+        nonNullable: false
+      isDynamic:
+        name: isDynamic
+        label: True if the value of this property is dependent on other properties
+          (as provided in DynamicParameterPropertyCodes)
+        range: boolean
+        max: 1
+        nonNullable: true
+      maxExclusive:
+        name: maxExclusive
+        label: Maximum value of the property, exclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      maxInclusive:
+        name: maxInclusive
+        label: Maximum value of the property, inclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      methodOfMeasurement:
+        name: methodOfMeasurement
+        label: Description of the method of measurement
+        range: string
+        max: 1
+        nonNullable: false
+      minExclusive:
+        name: minExclusive
+        label: Minimum value of the property, exclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      minInclusive:
+        name: minInclusive
+        label: Minimum value of the property, inclusive
+        range: decimal
+        max: 1
+        nonNullable: false
+      pattern:
+        name: pattern
+        label: 'An XML Schema Regular expression for the property value. See for explanation:
+          https://www.regular-expressions.info/xml.html.'
+        range: string
+        max: 1
+        nonNullable: false
+      physicalQuantity:
+        name: physicalQuantity
+        label: The quantity in plain text
+        range: string
+        max: 1
+        nonNullable: false
+      propertyValueKind:
+        name: propertyValueKind
+        label: 'Indicates kind of value: Single, Range (2 values expected), List (multiple
+          values expected), Complex (use in combination with ConnectedProperties),
```

Comment:
Gotta check, there are a lot of \r\n in the JSON which seem to be transfered in the YAML

Reply:
1. if newlines are significant in output, then don't introduce new newlines
2. for non-significant, I prefer NO newlines, because it's easier to carry
  `descr: a very long descr`
to a new place, ratjer than
```
descr:
  a very 
  long 
  descr
```